services/fill_file.rb

```diff
@@ -0,0 +1,11 @@
+class FillFile
+
+  def call(path)
+    puts I18n.t(:enter_words)
+    words = gets.chomp.split(',')
```

Comment:
Add here REGEXP validation that input contains only words (no numbers or symbols whatsoever)